benchmark/url/legacy-vs-whatwg-url-searchparams-parse.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+const common = require('../common.js');
+const { URLSearchParams } = require('url');
+const querystring = require('querystring');
+
+const inputs = {
+  noencode: 'foo=bar&baz=quux&xyzzy=thud',
+  encodemany: '%66%6F%6F=bar&%62%61%7A=quux&xyzzy=%74h%75d',
```

Comment:
I see, but I think it's beneficial to have a benchmark case with duplicated `&`, even w/o custom separators (i.e. #10454).